src/Form/Form.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import Input from '../Input/Input';
+import Checkbox from '../Checkbox/Checkbox';
+import '../style.css';
+
+class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      first: '',
+      last: '',
+      password: '',
+      checked: true
+    }
+  }
+
+  handleEmailChange = (event) => {
+    this.setState({email: event.target.value});
```

Comment:
Let's have one function handling the values of inputs, there's no need for 4 of them